.github/CODEOWNERS

```diff
@@ -1423,10 +1423,77 @@ x-pack/test/api_integration/deployment_agnostic/services/ @elastic/appex-qa
+/x-pack/test/functional/page_objects/status_page.ts @elastic/kibana-core
+/x-pack/test/functional/page_objects/share_saved_objects_to_space_page.ts @elastic/kibana-core
+/x-pack/test/functional/page_objects/banners_page.ts @elastic/kibana-core
+/x-pack/test/common/lib/test_data_loader.ts @elastic/kibana-core
+/x-pack/test/api_integration/services/usage_api.ts @elastic/kibana-core
+/x-pack/test/api_integration/apis/kibana @elastic/kibana-core
+/test/api_integration/fixtures/import.ndjson @elastic/kibana-core
+/x-pack/test/plugin_api_integration @elastic/kibana-core # Assigned per https://github.com/elastic/kibana/pull/146704
+/x-pack/test/localization/ @elastic/kibana-core # Assigned per https://github.com/elastic/kibana/pull/146704
+/test/ui_capabilities/newsfeed_err @elastic/kibana-core # Assigned per https://github.com/elastic/kibana/pull/66562
+/test/server_integration/services/types.d.ts @elastic/kibana-core # Assigned per https://github.com/elastic/kibana/pull/81140
+/test/server_integration/http @elastic/kibana-core
+/test/scripts/run_multiple_kibana_nodes.sh @elastic/kibana-core
+/test/plugin_functional/test_suites/application_links @elastic/kibana-core
+/test/interpreter_functional/test_suites/run_pipeline @elastic/kibana-core
+/test/functional/services/usage_collection.ts @elastic/kibana-core
+/test/functional/services/saved_objects_finder.ts @elastic/kibana-core
+/test/api_integration/fixtures/import_managed* @elastic/kibana-core
+/test/functional/services/apps_menu.ts @elastic/kibana-core
+/x-pack/test/functional/apps/status_page @elastic/kibana-core
+/x-pack/test/cloud_integration @elastic/kibana-core
 /x-pack/test/cloud_integration/plugins/saml_provider @elastic/kibana-core
-/x-pack/test/functional_embedded/plugins/iframe_embedded @elastic/kibana-core
+/test/server_integration/plugins/status_plugin_* @elastic/kibana-core
+/x-pack/test/functional_cors @elastic/kibana-core
+/x-pack/test/stack_functional_integration/apps/telemetry @elastic/kibana-core
+/test/plugin_functional/test_suites/telemetry @elastic/kibana-core
+/test/plugin_functional/test_suites/usage_collection @elastic/kibana-core
+/test/plugin_functional/test_suites/saved_objects* @elastic/kibana-core
+/test/plugin_functional/test_suites/core* @elastic/kibana-core
+/test/plugin_functional/plugins/telemetry @elastic/kibana-core
+/test/plugin_functional/plugins/session_notifications @elastic/kibana-core
+/test/plugin_functional/plugins/kbn_top_nav/ @elastic/kibana-core
+/test/plugin_functional/plugins/elasticsearch_client_plugin @elastic/kibana-core
+/test/plugin_functional/plugins/app_link_test @elastic/kibana-core
+/x-pack/test/plugin_functional/plugins/global_search_test @elastic/kibana-core
+/test/plugin_functional/plugins/saved_object* @elastic/kibana-core
+/test/plugin_functional/plugins/rendering_plugin @elastic/kibana-core
+/test/interpreter_functional/plugins/kbn_tp_run_pipeline @elastic/kibana-core
```

Comment:
I _think_ interpreter stuff is owned by `@elastic/kibana-visualizations`

Reply:
Jean-Louis, I think it's core: https://github.com/wayneseymour/kibana/blob/assign-core/test/interpreter_functional/plugins/kbn_tp_run_pipeline/kibana.jsonc#L4

Thanks for all your help on this btw!

Reply:
Based on this line in .CODEOWNERS:

```
packages/kbn-interpreter @elastic/kibana-visualizations
```

It made me think that ownership has changed over time to `@elastic/kibana-visualizations` even though someone on Core may have created this plugin. But I see the tests themselves are also edited a lot by `@elastic/appex-sharedux`. Happy to leave it under core for now as we'll need to update kibana.jsonc too.

Reply:
Ok thanks Jean-Louis.  
FWIW, there are going to be many ownership changes due to this project I'm working on, and due to the KBN Sustaainable Arch effort.